### DISCUSSION TITLE
Forex Rates Historical Data

### DISCUSSION THREAD

**Discussion by DaiMan00** [alvarobartt/investpy#480] [Q&A]
Can investpy allow users to download exchange rates data? 

Like USD/JPY from 01/01/2020 to 31/12/2020?

**Comment by anjan-cbi** [comment]
Hello @DaiMan00 you can use this

inv.get_currency_cross_historical_data(
currency_cross='USD/JPY',
from_date='01/01/2019',
to_date='31/12/2021'
)

![image](https://user-images.githubusercontent.com/50775503/147560572-bf5cd8f2-d6c9-4732-b6cc-3dbca65f2e57.png)
